pkg/actuators/client/client.go

```diff
@@ -66,3 +69,43 @@ func (c *ibmCloudClient) InstanceGet(instanceID string) (*vpcv1.Instance, error)
 	instance, _, err := c.vpcService.GetInstance(options)
 	return instance, err
 }
+
+// CustomImageGet retrieves custom image from VPC by region and name
+func (c *ibmCloudClient) CustomImageGetID(imageName string, regionName string) (*vpcv1.Image, error) {
+	// Get region info
+	region, _, err := c.vpcService.GetRegion(c.vpcService.NewGetRegionOptions(regionName))
```

Comment:
Would it be possible to have the VPC Service region set in the `NewClient()` constructor to avoid making this `GetRegion()` call each time `CustomImageGetID()` is called? Is it valid to create `Machines` on a cluster in a different region from what the cluster `Infrastructure` indicates?

Reply:
I thought about the same thing, but since we can have different regions in `MachineSet, Machine` I thought of leaving it here.